docs/upgrade-guide/upgrade411-50.md

```diff
@@ -0,0 +1,114 @@
+---
+Title: Upgrading from ADF v4.11 to v5.0
+---
+
+# Upgrading from ADF v4.11 to v5.0
+
+This guide explains how to upgrade your ADF v4.11 project to work with v5.0.
+
+Do not skip this task, if you want your application to be updated to a most recent version of ADF. 
+Upgrades of multiple versions of ADF cannot be done in one step only, but should follow the chain of sequential updates. 
+
+**Note:** the steps described below might involve making changes
+to your code. If you are working with a versioning system then you should
+commit any changes you are currently working on. If you aren't using versioning
+then be sure to make a backup copy of your project before going ahead with the
+upgrade.
+
+## Library updates
+
+### Automatic update using the Yeoman Generator
```

Comment:
yeoman generator does not perform any updates of the applications, it only generates new project scaffolds You don't need documenting how to create a yeoman app for people that want to upgrade the ADF

Reply:
Okay. Removed it.